src/app/modules/explore/explore.component.html

```diff
@@ -28,6 +28,11 @@
           ><button mat-button (click)="onGetPosts()"><mat-icon>settings</mat-icon>Get Posts</button
           ></mat-list-item
         >
+
+        <mat-list-item role="listitem"
+          ><button mat-button (click)="openNewPostDialog()">New post</button>
```

Comment:
for naming i suggest something like `onNewPost()` just to have a consistent schema